tests/Kernel/Paragraphs/ParagraphsTest.php

```diff
@@ -260,63 +256,85 @@ public function testListItem(): void {
     $html = $this->renderParagraph($paragraph);
     $crawler = new Crawler($html);
 
-    $this->assertCount(1, $crawler->filter('.ecl-list-item.ecl-list-item--thumbnail'));
-    $this->assertEquals('Item title', trim($crawler->filter('.ecl-list-item__title')->text()));
-    $this->assertEquals('Item description', trim($crawler->filter('.ecl-list-item__detail')->text()));
+    $this->assertEquals('Item title', trim($crawler->filter('article.ecl-content-item div.ecl-content-item__title')->text()));
+    $this->assertEquals('Item description', trim($crawler->filter('article.ecl-content-item div.ecl-content-item__description')->text()));
 
-    $link_element = $crawler->filter('.ecl-list-item__link');
+    $link_element = $crawler->filter('article.ecl-content-item div.ecl-content-item__title a.ecl-link');
     $this->assertCount(1, $link_element);
     $this->assertEquals('http://www.example.com/', $link_element->attr('href'));
 
-    $image_element = $crawler->filter('.ecl-list-item__secondary img.ecl-image');
+    $this->assertCount(3, $crawler->filter('article.ecl-content-item > div'));
+
+    $image_element = $crawler->filter('article.ecl-content-item > div[role="img"].ecl-u-d-lg-block');
     $this->assertCount(1, $image_element);
-    $this->assertEquals(
+    $this->assertContains(
       file_url_transform_relative(file_create_url($image->getFileUri())),
-      $image_element->attr('src')
+      $image_element->attr('style')
     );
-    $this->assertEquals('Druplicon', $image_element->attr('alt'));
+    $this->assertEquals('Druplicon', $image_element->attr('aria-label'));
 
-    $meta_elements = $crawler->filter('.ecl-meta__item');
-    $this->assertCount(3, $meta_elements);
-    $this->assertEquals('Meta 1', trim($meta_elements->getNode(0)->nodeValue));
-    $this->assertEquals('Meta 2', trim($meta_elements->getNode(1)->nodeValue));
-    $this->assertEquals('Meta 3', trim($meta_elements->getNode(2)->nodeValue));
+    $this->assertEquals('Meta 1 | Meta 2 | Meta 3', trim($crawler->filter('article.ecl-content-item div.ecl-content-item__meta')->text()));
 
-    // The primary image markup should not be rendered.
-    $this->assertCount(0, $crawler->filter('.ecl-list-item__primary img.ecl-image'));
     // The date field should not be rendered.
     $this->assertCount(0, $crawler->filter('.ecl-date-block__week-day'));
     $this->assertCount(0, $crawler->filter('.ecl-date-block__day'));
     $this->assertCount(0, $crawler->filter('.ecl-date-block__month'));
```

Comment:
Since that markup will never be there these 3 assertions will always be true, I suggest to comment them out and add a todo on top referring to the date variant issue, so we make sure we deal with this particular thing in the scope of that issue.

Reply:
Perfect.